txpool/txpool.go

```diff
@@ -265,6 +265,11 @@ func (t *TxPool) GetNonce(addr types.Address) (uint64, bool) {
 	return highestNonce + 1, true
 }
 
+// GetCapacity returns the current state of the gauge
```

Comment:
Please document the exact data that this method returns

Reply:
[Changed](https://github.com/0xPolygon/polygon-sdk/pull/262/commits/0f68e9da1b67f49eed2125d022dad4b4c7953b75)